src/dispatch/incident_cost/scheduled.py

```diff
@@ -0,0 +1,55 @@
+import logging
+
+from schedule import every
+
+from dispatch import service as incident_service
+from dispatch.decorators import background_task
+from dispatch.incident_cost_type import service as incident_cost_type_service
+from dispatch.scheduler import scheduler
+
+from .service import (
+    calculate_incident_opportunity_cost,
+    get_by_incident_id_and_incident_cost_type_id,
+)
+
+
+log = logging.getLogger(__name__)
+
+
+@scheduler.add(every(5).minutes, name="calculate-incidents-opportunity-cost")
+@background_task
+def calculate_incidents_opportunity_cost(db_session=None):
+    """
+    Calculates and saves the opportunity cost for all incidents.
+    """
+    opportunity_cost_type = incident_cost_type_service.get_by_name("Opportunity Cost", db_session)
```

Comment:
We can ship Dispatch with an incident cost type called `Opportunity Cost` or perhaps allow to set a default cost type and use that here.

Reply:
Yeah, I would prefer we follow setting defaults as we do with incident types.